subprojects/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/TransformOperationMapper.java

```diff
@@ -44,7 +44,7 @@
  *
  * @since 5.1
  */
-class TransformOperationMapper implements BuildOperationMapper<ExecuteScheduledTransformationStepBuildOperationDetails, DefaultTransformDescriptor>, OperationDependencyLookup {
+class TransformOperationMapper implements BuildOperationMapper<ExecutePlannedTransformStepBuildOperationDetails, DefaultTransformDescriptor>, OperationDependencyLookup {
```

Comment:
```suggestion
class PlannedTransformStepOperationMapper implements BuildOperationMapper<ExecutePlannedTransformStepBuildOperationDetails, DefaultTransformDescriptor>, OperationDependencyLookup {
```

🤔 Should this mapper now be called different?

Reply:
Potentially, but I would do the rename in a separate PR